addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -340,50 +359,45 @@ def _plan_filter_activity_templates_to_schedule(self):
 
     def _get_summary_lines(self, templates):
         self.ensure_one()
-        summaries = []
+        summaries = self.env['mail.activity.schedule.summary']
         for template in templates:
             activity_type = template.activity_type_id
-            summary_line = activity_type.name
-            if template.summary:
-                summary_line += f": {template.summary}"
-            # We don't display deadlines when the user doesn't specify a plan_date
+            summary_line = summaries.new({'schedule_id': self})
```

Comment:
You don't need to give a schedule_id. It is a o2m, using standard commands will automatically populate it. You only have to give line specific values (summary, responsible, ...).